bftclient/src/msg_receiver.cpp

```diff
@@ -0,0 +1,100 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use
+// this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license
+// terms. Your use of these subcomponents is subject to the terms and conditions of the
+// subcomponent's license, as noted in the LICENSE file.
+
+#include "kvstream.h"
+#include "assertUtils.hpp"
+#include "bftengine/ClientMsgs.hpp"
+#include "msg_receiver.h"
+
+namespace bft::client {
+
+void UnmatchedReplyQueue::push(UnmatchedReply&& reply) {
```

Comment:
Looks like you finally moved to a low-case function names :-)

Reply:
camelCase is now official :)